README.md

```diff
@@ -19,14 +19,21 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
 
+[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
+
+
+### Test Harnesses
+
 [iOS Test Harness](https://github.com/realeyes-media/scte35-decoder-multiplatform-iOS-harness)
 
-[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
 
 ### How to Include on Your Platform
 
 
-**Javascript**
+
+## Javascript
```

Comment:
It's odd that the subheadings under "How to Include on Your Platform" are given a bigger Heading status. Using 4 "#"s might be more appropriate.